src/com/owncloud/android/lib/resources/status/OCCapability.java

```diff
@@ -294,4 +305,5 @@ public CapabilityBooleanType getFilesVersioning() {
     public void setFilesVersioning(CapabilityBooleanType filesVersioning) {
         this.mFilesVersioning = filesVersioning;
     }
+
```

Comment:
IMHO, this comment is over-zealous. The value of removing this line break is 0, and is not worth the time needed to do it.